google/cloud/spanner_v1/database.py

```diff
@@ -104,6 +112,13 @@ def __init__(self, database_id, instance, ddl_statements=(), pool=None):
         self._create_time = None
         self._restore_info = None
 
+        if type(encryption_config) == dict:
+            self._encryption_config = EncryptionConfig(
+                kms_key_name=encryption_config["kms_key_name"]
+            )
+        else:
```

Comment:
Would it happen that a user passes a `str` ("kms_key") into this function? From what I can see, `encryption_config` only allows two types: `dict` and `EncryptionConfig`. Should we say if type is not a `EncryptionConfig`, then we should raise an InvalidArgument error?

Reply:
I've added some documentation explaining the permitted formats of the `encryption_config`. I have not added a TypeError as it is inconsistent with other similar conversions in the library which do not raise errors. However, I may look into adding these errors in a future PR.